internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -156,6 +156,51 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				m, _ := json.Marshal(jsonPayload)
 				json.Unmarshal(m, &jsonStructData)
 			}
+		case "cloudflare_access_policy":
+			if accountID != "" {
+				applications, _, err := api.AccessApplications(context.Background(), accountID, cloudflare.PaginationOptions{})
```

Comment:
we don't yet handle business logic dependencies like this and probably don't want to manually do this for each resource that has them. i'd remove this for now and manually add the application IDs until we have a maintainable fix for dependant resources.